crates/api/src/runtime.rs

```diff
@@ -385,6 +386,182 @@ impl Config {
         self.memory_creator = Some(MemoryCreatorProxy { mem_creator });
         self
     }
+
+    /// Configures the maximum size where a linear memory is considered static,
```

Comment:
Because wasm sometimes measures sizes in pages, we should clarify that this is a size in bytes.